draft-wang-ppm-differential-privacy.md

```diff
@@ -189,18 +189,18 @@ Noise can be introduced at various steps at the computation, and by various
 parties. Depending on the mechanism: the Clients might add noise to their own
 measurements; the Aggregators might add noise to their aggregate shares (the
 values they produce for the Collector); and the Collector might add noise to
-the aggregate result before publishing it. We refer to these classes of
-mechanisms as "Client-DP", "Aggregator-DP", and "Collector-DP" respectively.
-[TODO: I put "Collector-DP" here as a strawman. Would we actually want to use
-it?]
+the aggregate result before publishing it.
 
-In addition, it is possible to compuse different DP mechanisms. [TODO: Say why
-we would want to do this and give an illustrative example of combining
-Client-DP and Aggregator-DP.]
+> NOTE The Collector adding noise is mentioned here only as a strawman. We
+> typically consider the Collector to be under control of the attacker, so we
+> want to ensure the aggregate result is DP before the Collector computes it.
 
-[TODO: Transition text] This document defines a "DP policy" as the composition
-of an optional Client-DP mechanism and an optional Aggregator-DP mechanism
-with a VDAF.
+For some DP mechanisms, noise is added only by the Clients or only by the
```

Comment:
mechanisms -> policies? If we are talking about noise by multiple parties here.

Reply:
Good catch. I also added a definition of "DP policy" above.